.github/workflows/sycl_nightly.yml

```diff
@@ -48,6 +52,42 @@ jobs:
       retention-days: 90
       lts_matrix: ${{ needs.test_matrix.outputs.lts_wn_matrix }}
 
+      # We use the build for the release, override its name.
+      artifact_archive_name: sycl_windows.tar.gz
+
+  nightly_release:
```

Comment:
Let's use "build" instead of "release" everywhere to avoid confusion about quality guarantees.

Reply:
I don't think those are user visible. "Build" would also be confusing for the CI developers as technically we are doing "Github release" here and not building anything.

Reply:
Can we meet in the middle with "build_prerelease" ?

Reply:
Why do you want to call it prerealse? 
Is it just because GitHub doesn't allow anything other than "release" and "pre-release"?
If so, I don't see much sense in calling our nightly builds "pre-release".

Reply:
> I don't think those are user visible. 

The next line defines user visible name. 

> "Build" would also be confusing for the CI developers as technically we are doing "Github release" here and not building anything.

"Compiler build upload"?